Add stealing a meeting room (#8)
clearhaus/cake-legislation: legislation.tex

  \item Forgetting that you are on call.
  \item Being made (partially) superfluous by automation.
  \item Loosing office key fob.

>   \item Occupy a meeting room, which was booked by another person

Reviewer: ```suggestion
  \item Occupying a meeting room booked by another person (without leaving immediately)
```
